### PR TITLE
Add parameter to get user/public genesets (#87).

### PR DESCRIPTION
**Pull request by everaldorodrigo**
### Context

Issue: #87

### CODE REVIEW

File: src/tests/test_remote.py [hunk 1]
```diff
@@ -77,6 +77,54 @@ def test_query_by_source_smpdb(self):
...
+    def test_include_curated_filter_plus_query(self):
+        query = self.query(q='glucose', species='9615', include='curated')
+        assert query['hits'][0]['taxid'] == "9615"
+
+    def test_include_public_filter_plus_query(self):
+        query = self.query(q='glucose', species='9615', include='public')
+        assert query['hits'][0]['taxid'] == "9615"
+
+    def test_include_user_filter_plus_query(self):
+        res = self.request("query?q=glucose&species=9615&include=user").json()
```
Comment (everaldorodrigo) [inline src/tests/test_remote.py:103]: The method `self.query(q='glucose', species='9615', include='user')` is raising an error when content is not found. Is that correct? Same for the next test called `test_include_anonymous_filter_plus_query`.

Comment (newgene) [reply src/tests/test_remote.py:103]: that should be fine.

File: src/web/pipeline.py [hunk 2]
```diff
@@ -45,4 +44,14 @@ def apply_extras(self, search, options):
             if options.aggs and options.source_facet_filter:
                 search = search.post_filter('terms', source=options.source_facet_filter)
 
+        if options.include:
+            if 'public' == options.include:
```
Comment (newgene) [inline src/web/pipeline.py:48]: somehow I feel `if options.include == "public"` is more natural for me

File: src/tests/test_remote.py [hunk 3]
```diff
@@ -14,68 +15,110 @@ class TestMyGenesetDataIntegrity(MyGenesetWebTestBase):
...
+        assert len(res["hits"]) == 0
+
+    def test_include_anonymous_filter_plus_query(self):
+        res = self.request("query?q=glucose&species=9615&include=anonymous").json()
+        assert res["total"] == 0
+        assert len(res["hits"]) == 0
+
+    def test_include_filter_error(self):
+        res = self.request("query?include=a_wrong_include", expect=400).json()
+        assert res == {
```
Comment (everaldorodrigo) [inline src/tests/test_remote.py:115]: WDYT about using map/dict in the assert?

Comment (newgene) [reply src/tests/test_remote.py:115]: looks fine to me. If later you find any field could change, you can skip that field.

**Pull request merged**